torch/jit/__init__.py

```diff
@@ -813,6 +814,9 @@ def __getitem__(self, k):
     def __setitem__(self, k, v):
         raise NotImplementedError
 
+    def copy(self):
+        return self.__class__(self)
```

Comment:
Shouldn't this be `self.__class__(self.submodule.get())`?

Reply:
> Shouldn't this be `self.__class__(self.submodule.get())`?

@driazati Thanks for your comment. I'm the author of #15421 . I tried `self.__class__(self.submodule.get())` but got `AttributeError: 'OrderedParameterDict' object has no attribute 'submodule'`